src/expandableCalendar/agendaList.js

```diff
@@ -146,8 +146,9 @@ class AgendaList extends Component {
         onScroll={this.onScroll}
         onMomentumScrollBegin={this.onMomentumScrollBegin}
         onMomentumScrollEnd={this.onMomentumScrollEnd}
-        // onScrollToIndexFailed={(info) => { console.warn('onScrollToIndexFailed info: ', info); }}
-        // getItemLayout={this.getItemLayout} // onViewableItemsChanged is not updated when list scrolls!!!
+        stickySectionHeadersEnabled
```

Comment:
Spoke offline. If we move the prop expansion under here then we should be able to override all of these at will. Including stickySectionHeaders

Reply:
This should solve the custom render props as well. I will move it.